.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -234,9 +234,14 @@ run_tests() {
 
   droute_send $release_name $project
 
-  exit ${RESULT}
+  echo ${RESULT}
+
+  if [ ${RESULT} -ne 0 ]; then
+    exit ${RESULT}
+  fi
```

Comment:
@gustavolira sure,

At the top of `openshift-ci-tests.sh` define
`OVERALL_RESULT=0`
```suggestion
  echo "$project RESULT: $RESULT"

  if [ ${RESULT} -ne 0 ]; then
    OVERALL_RESULT=1
  fi
```

then at the end of main() you can exit with: 
```
 initiate_deployments
 check_and_test ${RELEASE_NAME} ${NAME_SPACE}
 check_and_test ${RELEASE_NAME_RBAC} ${NAME_SPACE_RBAC}
 exit $OVERALL_RESULT
```

also update the `check_and_test()` as : 
```
check_and_test() {
  local release_name=$1
  local namespace=$2
  check_backstage_running $release_name $namespace
  backstage_status=$?
  echo "Display pods for verification..."
  oc get pods -n $namespace
  if [ $backstage_status -ne 0 ]; then
    echo "Backstage is not running. Exiting..."
    OVERALL_RESULT=1
  fi
  run_tests $release_name $namespace
}
```